packages/compass-aggregations/src/modules/pipeline-builder/stage-editor.ts

```diff
@@ -417,19 +416,11 @@ const ESCAPED_STAGE_OPERATORS = STAGE_OPERATORS.map((stage) => {
 
 function getStageSnippet(
   stageOperator: string | null,
-  env: string,
   shouldAddComment: boolean,
   escaped = false
 ) {
   const stage = (escaped ? ESCAPED_STAGE_OPERATORS : STAGE_OPERATORS).find(
-    (stageOp) => {
-      return (
-        stageOp.value === stageOperator &&
-        (stageOp.env as readonly typeof ENVS[number][]).includes(
-          env as typeof ENVS[number]
-        )
-      );
-    }
+    (stageOp) => stageOp.value === stageOperator
```

Comment:
For "duplicated" stages that have different snippets based on the env, should the env still be taken into account somehow here?

Reply:
Yes, that makes sense. Should have done it that way.